cmd/dockerd/daemon.go

```diff
@@ -520,6 +521,16 @@ func loadDaemonCliConfig(opts *daemonOptions) (*config.Config, error) {
 	conf.LogLevel = opts.LogLevel
 	conf.LogFormat = log.OutputFormat(opts.LogFormat)
 
+	if ver := os.Getenv("DOCKER_MIN_API_VERSION"); ver != "" {
+		if versions.LessThan(ver, api.MinVersion) {
+			return nil, errors.Errorf(`invalid DOCKER_MIN_API_VERSION: minimum supported API version is %s: %s`, api.MinVersion, ver)
```

Comment:
Discussed in maintainers call; let's remove / un-export api.MinVersion, so that users don't depend on it

Reply:
done; removed from api and moved to cmd/dockerd as non-exported const.